examples/data_connectors/NotionDemo.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
the jupyter notebook seems to run into issues? does it run?

Reply:
It was running fine for me? What did you give it for database_id param?